package.json

```diff
@@ -26,7 +26,6 @@
     "size:debug": "npm run build && size-limit --why",
     "start-js": "node scripts/start.js",
     "start": "node scripts/start.js",
-    "std-version:release": "standard-version -m \"chore(release): version %s\"",
     "std-version": "standard-version -m \"chore(release): version %s build ${TRAVIS_BUILD_NUMBER} [ci skip]\"",
```

Comment:
so run `/ci-scripts/publish_release.sh' to release?

Reply:
Yeah, `npm run release` calls `/ci-scripts/publish_release.sh`, which calls `/ci-scripts/publish.sh` inside of travis so that we have the correct variables. We can reuse the same `npm run std-version` now that we don't need a `[ci-skip]` and non-`[ci-skip]`